misc/chip_data.json

```diff
@@ -192,7 +192,7 @@
         "5858a52e-54fb-4bb2-80a0-ec5fce88b3ef": {
           "Name": "AI Has Line of Sight To Target",
           "ReadonlyTypeParams": {
-            "T": "(AI | Combatant | CreationObject | PatrolPoint | Player | Vector3)"
+            "T": "(AI | Combatant | CustomObject | PatrolPoint | Player | Vector3)"
```

Comment:
These fields that are prefixed with "Readonly" don't actually get read back into the game. However, I think we are doing a naming pass on them internally so you should see them change in the future.

Reply:
No need to change these back since they will just get discarded but more of an FYI for you.

Reply:
Actually, on second thought, these will be much easier for us to integrate if we can git rid of the change of `CreationObject` to `CustomObject`. I think that's the only major thing I see right now. If you do that I can read through the descriptions and see if we can get them in.